package.json

```diff
@@ -33,6 +33,7 @@
   },
   "homepage": "https://github.com/erichbehrens/react-animated-slider#readme",
   "dependencies": {
+    "path": "^0.12.7",
```

Comment:
Where is this used?

Reply:
its not and it doesn't appear on my branch (not sure where it came from!), so if you want to remove then feel free.